File: .Scripts/PrintTODO.lua
#!/bin/lua5.4

path  = "/home/mario/.local/share/todo/todo.txt"

file = io.open(path)
tasks  = {}
number = 1
for line in file:lines() do
    tasks[#tasks + 1] = "    \27[1m"..number.."\27[0m "..line
    number = number + 1
end
file:close()
for i,line in ipairs(tasks) do
    print(line)
end 

File: .Scripts/PrintTODO.pl
#!/bin/perl

use Term::ANSIColor;

my $path = "/home/mario/.local/share/todo/todo.txt";
my $number = 1;
open(my $file, "<", $path);
my @tasks = <$file>;
foreach(@tasks) {
    print color ('bold');
    print("    $number:");
    print color ('reset');
    print(" $_");
    $number = $number + 1;
}
